### PR TITLE
Adding GetConn method to return the beanstalk connection

### PR DESCRIPTION
**Pull request by NevilleC**
As it might be useful.

### CODE REVIEW

File: beanstalkworker/rawJob.go [hunk 1]
```diff
@@ -88,6 +88,11 @@ func (job *RawJob) GetTube() string {
 	return job.tube
 }
 
+// GetConn returns the beanstalk connection
```
Comment (tomponline) [inline beanstalkworker/rawJob.go]: thanks @NevilleC can you slightly change this comment, other than that its good to go.

```// GetConn returns the beankstalk connection used to receive this job.```

Added full stop to end and clarified that this is the connection for the current job (As I am planning to modify bsworker to use multiple connections and concurrent workers).

Comment (tomponline) [reply beanstalkworker/rawJob.go]: @NevilleC can you also update the demo app to use this function in some way - that way the demo can use all of the features.

Comment (NevilleC) [reply beanstalkworker/rawJob.go]: Sure @tomponline.

File: beanstalkworker/rawJob.go [hunk 2]
```diff
@@ -88,6 +88,11 @@ func (job *RawJob) GetTube() string {
 	return job.tube
 }
 
+// GetConn returns the beakstalk connection used to receive the job.
```
Comment (tomponline) [inline beanstalkworker/rawJob.go]: @NevilleC i made a typo in my suggestion, which you copied :( "beakstalk" :(

**Comment by tomponline** [comment]
cheers @NevilleC if you could update the demo app to show its usage then I'll merge :)

**Pull request merged**